File: src/test/java/io/bloco/faker/components/TeamTest.java
package io.bloco.faker.components;

import org.junit.Before;
import org.junit.Test;

import io.bloco.faker.Faker;

import static org.junit.Assert.assertNotNull;

public class TeamTest {

    private Faker faker;

    @Before
    public void setUp() {
        faker = new Faker();
    }

    @Test
    public void name() {
        assertNotNull(faker.team.name());
    }

    @Test
    public void creature() {
        assertNotNull(faker.team.creature());
    }

    @Test
    public void state() {
        assertNotNull(faker.team.state());
    }

    @Test
    public void sport() {
        assertNotNull(faker.team.sport());
    }
}

File: src/test/kotlin/io/bloco/faker/components/TeamTest.kt
package io.bloco.faker.components

import io.bloco.faker.Faker
import junit.framework.TestCase.assertNotNull
import org.junit.Test

class TeamTest {
    private val faker = Faker()

    @Test
    fun name() {
        assertNotNull(faker.team.name())
    }

    @Test
    fun creature() {
        assertNotNull(faker.team.creature())
    }

    @Test
    fun state() {
        assertNotNull(faker.team.state())
    }

    @Test
    fun sport() {
        assertNotNull(faker.team.sport())
    }
}
